Sources/Extensions/UIKit/UIImageExtensions.swift

```diff
@@ -73,7 +73,7 @@ public extension UIImage {
 	public func scaled(toHeight: CGFloat, with orientation: UIImageOrientation? = nil) -> UIImage? {
 		let scale = toHeight / size.height
 		let newWidth = size.width * scale
-		UIGraphicsBeginImageContext(CGSize(width: newWidth, height: toHeight))
+        UIGraphicsBeginImageContextWithOptions(CGSize(width: newWidth, height: toHeight), false, scale)
```

Comment:
It would be great if we could check for setting the value of `opaque`. You've put it to `false` by default, but we can potentially improve it. [This SO question](https://stackoverflow.com/questions/38999305/issue-in-checking-uiimage-have-alpha-transparant-color-or-not) seems to suggest that we can get the answer. Do you mind having a look?

Reply:
Could also just add a parameter to let the user control this flag